src/main/java/seedu/address/model/issue/UniqueIssueList.java

```diff
@@ -68,6 +68,34 @@ public void setIssue(Issue target, Issue editedIssue) {
         internalList.set(index, editedIssue);
     }
 
+    public Issue getIssue(IssueId id) {
+        requireAllNonNull(id);
+        for (Issue i: internalList) {
+            if (i.getIssueId() == id) {
```

Comment:
`==` may cause some unforeseen bugs later on, `.equals()` might be a better choice